deploy/manifests/dev/us-east-2/tenant/storetheindex/dhstore-cluster/base/dhstore/pvc.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: dhstore-data
+spec:
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: 1Ti
```

Comment:
1Ti might be not enough (even though we do have 10 instances). Right now we are already at 18(ish)Ti  in prod. Shall we bump that up to 2Ti at least?